py2puml/inspection/inspectclass.py

```diff
@@ -119,34 +78,42 @@ def parse_class_type(
                     attr_type = f"{composition_rel}[{', '.join(component_names)}]"
                 else:
                     attr_type = attr_raw_type
-            uml_attr = UmlAttribute(attr_name, attr_type)
+            uml_attr = UmlAttribute(attr_name, attr_type, True)
             definition_attrs.append(uml_attr)
 
-    handle_inheritance_relation(class_type, class_type_fqdn, root_module_name, domain_relations)
-
+    return definition_attrs
 
-def parse_type(
-    definition_type: Type,
+def inspect_class_type(
+    class_type: Type,
+    class_type_fqdn: str,
```

Comment:
Does the `d` in `fqdn` stands for "domain"? Probably, using `m` for "module" is more correct here?

Reply:
`fqdn` stands for "fully qualified domain name", a term that must be used for URLs. I got confused with the UML "fully qualified name" expression. I am fixing that too all over the codebase